koku/api/report/test/util/constants.py

```diff
@@ -78,6 +89,9 @@ def __setitem__(self, item, value):
         "resource_ids": AWS_RESOURCE_IDS,
         "units": AWS_UNITS,
         "cost_category": AWS_COST_CATEGORIES,
+        "operating_systems": AWS_OPERATING_SYSTEMS,
+        "vcpus": AWS_VCPUS,
```

Comment:
Maybe change it to "vcpu", in singular? Just to keep consistent with the api response?

Reply:
using plural here because we are referring multiple options